--- po4a-0.44/Po4aBuilder.pm.old	2013-07-29 17:13:54.974419735 +0100
+++ po4a-0.44/Po4aBuilder.pm	2013-07-29 17:14:08.312408777 +0100
@@ -166,7 +166,7 @@
     $self->depends_on('code');
     $self->make_files_writable("po/pod");
 
-    my $cmd = "PERL5LIB=lib perl po4a "; # Use this version of po4a
+    my $cmd = "PERL5LIB=lib perl ./po4a "; # Use this version of po4a
     $cmd .= "--previous ";
     $cmd .= "--no-translations ";
     $cmd .= "--msgid-bugs-address po4a-devel\@lists.alioth.debian.org ";
@@ -190,7 +190,7 @@
     $options{utf8} = 1;
     my $parser = Pod::Man->new (%options);
 
-    my $cmd = "PERL5LIB=lib perl po4a "; # Use this version of po4a
+    my $cmd = "PERL5LIB=lib perl ./po4a "; # Use this version of po4a
     $cmd .= $ENV{PO4AFLAGS}." " if defined($ENV{PO4AFLAGS});
     $cmd .= "--previous po/pod.cfg";
     system($cmd) and die;
